### PR TITLE
Bump version to 0.29.0

### PR DESCRIPTION
**Pull request by sunboshan**

### CODE REVIEW

**Comment by tyrchen** [comment]
I happened to need to add a utility function for the ConnSupervisor. :)

**Comment by sunboshan** [comment]
Not a problem!!

File: lib/forge_sdk/conn/supervisor.ex [hunk 1]
```diff
@@ -34,6 +34,10 @@ defmodule ForgeSdk.ConnSupervisor do
     DynamicSupervisor.which_children(__MODULE__)
   end
 
+  def get_names do
+    Enum.map(children(), fn {_, p, _, _} -> Process.info(p)[:registered_name] end)
```
Comment (sunboshan) [inline lib/forge_sdk/conn/supervisor.ex]: Process.info(p) will give lots of info including gc and other stuff, it's better to use Process.info(p, :registered_name) to only get info that needed.

Comment (tyrchen) [reply lib/forge_sdk/conn/supervisor.ex]: thanks!

File: lib/forge_sdk/conn/supervisor.ex [hunk 2]
```diff
@@ -34,6 +34,10 @@ defmodule ForgeSdk.ConnSupervisor do
     DynamicSupervisor.which_children(__MODULE__)
   end
 
+  def get_names do
+    Enum.map(children(), fn {_, p, _, _} -> Process.info(p, :registered_name) end)
```
Comment (sunboshan) [inline lib/forge_sdk/conn/supervisor.ex]: iex(7)> Process.info(pid(0,0,0))[:registered_name]
:init

iex(6)> Process.info(pid(0,0,0),:registered_name)
{:registered_name, :init}

You probably want to do a pattern matching to get the name if this function is expecting to return a list of names.

**Pull request merged**